python/cudf/cudf/core/frame.py

```diff
@@ -2463,6 +2487,10 @@ def any(self, axis=0, skipna=True, level=None, **kwargs):
             **kwargs,
         )
 
+    # TODO: This is not a pandas reduction, it's something that cudf added. Do
+    # we want to continue supporting this? If so, for which types? It seems odd
```

Comment:
this appears to be supported all the way down into libcudf. It could still be useful to have around for something that uses it in a formula, but I don't see a reason to keep it exposed to users especially if we have to invent/support rules for its edge cases and whatnot ourselves. What if we keep it around as a private method?

Reply:
groupby-apply would obviate the need for this when it's available. I recall `sum_of_squares` being needed somewhere in the dask_cudf machinery in the past (cc: @rjzamora) but it looks like this is no longer the case. Even if it is, dask-cudf could move to using groupby-apply.

Let's deprecate, and remove in 23.04.

Reply:
Deprecated.